### PR TITLE
[spec] Add spec for collecting ansible_host

### PR DESCRIPTION
**Pull request by SteveHNH**
The client will begin collecting ansible_host and putting it into the
root of the archive. This spec will let us dig it out of puptoo for
system_profile

RHCLOUD-13536

Signed-off-by: Stephen Adams <tsadams@gmail.com>

### CODE REVIEW

**Comment by bfahr** [comment]
@SteveHNH I submitted #3008 for @gravitypriest but I'll close it and use this one instead.  It also needs to go through spec approval.

**Review by bfahr — APPROVED** [review]
Approved pending spec approval

File: insights/specs/insights_archive.py
Comment (bfahr) [inline insights/specs/insights_archive.py:16]: This change is not required, it will be covered by the `Core3Specs`.

**Review by bfahr — CHANGES_REQUESTED** [review]

**Comment by SteveHNH** [comment]
@bfahr all fixed up

File: insights/specs/core3_archive.py
Comment (bfahr) [inline insights/specs/core3_archive.py:17]: @SteveHNH leave this here, and add `ansible_host` here as well to support core processing of core-collection archives.

File: insights/specs/default.py
Comment (bfahr) [inline insights/specs/default.py:136]: Remove this spec and add it to `insights_archive.py` to support reading non-core-collection archives.

File: insights/specs/default.py
Comment (bfahr) [inline insights/specs/default.py:174]: Remove this spec, its already in `insights_archive.py`.

**Review by bfahr — CHANGES_REQUESTED** [review]

**Review by bfahr — APPROVED** [review]

**Pull request merged**
